charts/synthetics-private-location/templates/_helpers.tpl

```diff
@@ -40,9 +40,6 @@ helm.sh/chart: {{ include "synthetics-private-location.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- if .Values.commonLabels}}
-{{ toYaml .Values.commonLabels }}
-{{- end }}
```

Comment:
Shouldn't this be removed below, in the `Selector labels` part? 
https://github.com/DataDog/helm-charts/blob/07f38ede48cd01b7ba18000c622eccaeb143d035/charts/synthetics-private-location/templates/_helpers.tpl#L54-L56

Reply:
If we remove commonLabels from selector labels, they won't be used for [selectors](https://github.com/DataDog/helm-charts/blob/611690287a32951452dc0d9bc014d9ffb3265d58/charts/synthetics-private-location/templates/deployment.yaml#L11) anymore, so I think this PR is removing the right template block